src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1877,6 +1885,16 @@ protected bool MarkDefaultConstructor (TypeDefinition type, in DependencyInfo re
 			return MarkMethodIf (type.Methods, MethodDefinitionExtensions.IsDefaultConstructor, reason, sourceLocationMember) != null;
 		}
 
+		protected bool MarkGetInstanceMethod (TypeDefinition type, in DependencyInfo reason, IMemberDefinition sourceLocationMember)
```

Comment:
Nit: why return `bool`  (not used in here)... and also why `protected`.

Reply:
I copy-pasted MarkDefaultConstructor above. No special reason. Undone.